sound/soc/sof/intel/byt.c

```diff
@@ -86,606 +44,33 @@ static const struct snd_sof_debugfs_map byt_debugfs[] = {
 	 SOF_DEBUGFS_ACCESS_ALWAYS},
 };
 
-static const struct snd_sof_debugfs_map tng_debugfs[] = {
+static const struct snd_sof_debugfs_map cht_debugfs[] = {
 	{"dmac0", DSP_BAR, DMAC0_OFFSET, DMAC_SIZE,
 	 SOF_DEBUGFS_ACCESS_ALWAYS},
-	{"dmac1", DSP_BAR, DMAC1_OFFSET, DMAC_SIZE,
+	{"dmac1", DSP_BAR,  DMAC1_OFFSET, DMAC_SIZE,
 	 SOF_DEBUGFS_ACCESS_ALWAYS},
-	{"ssp0", DSP_BAR, SSP0_OFFSET, SSP_SIZE,
+	{"dmac2", DSP_BAR,  DMAC2_OFFSET, DMAC_SIZE,
```

Comment:
Here, too. There are double space

Reply:
thanks @bardliao, this wasn't intentional. will fix.

Reply:
fixed